.github/dependabot.yml

```
@@ -1,10 +1,14 @@
 version: 2
 updates:
-  - package-ecosystem: maven
-    directory: /
+  - package-ecosystem: docker
+    directory: /keycloak/src/main/docker
```

Is this prefix correct? I would expect `src/main/docker`.
```suggestion
    directory: src/main/docker
```